client/client.go

```diff
@@ -108,6 +109,11 @@ func (c *DatabricksClient) Do(ctx context.Context, method, path string,
 	opts = append(opts, httpclient.WithRequestHeaders(headers))
 	opts = append(opts, httpclient.WithRequestData(request))
 	opts = append(opts, httpclient.WithResponseUnmarshal(response))
+	// Remove extra `/` from path for files API.
```

Comment:
@mgyucht it's better applied in the `RequestVisitors` slice, as it'll be more forward-compatible. 

I wonder if we can trim+add a slash from end of URL and remove the slash from the beginning of the path? it'll be more logical and fit more cases

Reply:
This is at its core a big in the files API spec. I don't want to support this in general and will fix it soon, after which I'll remove it. So consider this change to be temporary until the next OpenAPI bump.

Reply:
let's better do a more proactive change https://github.com/databricks/databricks-sdk-go/pull/713